### ISSUE TITLE
Feature: Support Garagedoor Opener

### ISSUE THREAD

**Issue by chbloemer** [hap-java/openhab2-addons#4]
Is it currently possible to tag a Rollershutter as an Garagedoor Opener, so Homekit doesn't think its an WindowCovering? I saw a post in forum where you can see the Garagedoor.
My Garagedoor uses RollershutterTyp in Openhab.

https://community.openhab.org/t/announcing-some-improvements-to-the-homekit-plugin-for-openhab2/62729/84

I would like to prevent that my Garagedoor opens when I say: "Hey Siri, please open all my Blinds"

If not, the implementation should be similar to Window Covering.

**Comment by ccutrer** [comment]
No, not at this time. But yes, it should be simple to implement in the future.

**Comment by chbloemer** [comment]
Wow you almost answering in realtime. Awesome. 🤣

**Comment by danricho** [comment]
This would be great, especially with the new CarPlay improvements coming.

**Comment by Digitsl-dB** [comment]
CarPlay with iOS13 includes Siri suggestions such as suggesting to open the garage door when you arrive home on the car screen, this would be great.
Will there be support for Garage Door Opener in the next addon jar release?
